### DISCUSSION TITLE
AnvilLoader Problem

### DISCUSSION THREAD

**Discussion by Matevoo** [Minestom/Minestom#1655] [Help] [answered]
Hello I encountered an error where I had already created an instance while setting a world to it using AnvilLoader  - example code: 

```
public Instance createInstance() throws FileNotFoundException {
        System.out.println(returnFullPath().toString());
        if (!this.hasMapFile()) {
            throw new FileNotFoundException(
                    "Could not find instance folder for " + this.name
            );
        }
        return MinecraftServer.getInstanceManager().createInstanceContainer(
                new AnvilLoader(
                        returnFullPath()
                )
        );
    }
```

 - I can create that instance just fine using the AnvilLoader but when I try to set player's instance into that instance where was the world loaded I get this error: https://paste.learnspigot.com/avaguyazup.bash

**Comment by Distractic** [comment, marked as answer]
Try to load the world in solo with the same version than your minestom server version
Then, retry to load  it with AnvilLoader

**Reply by Matevoo** [reply]
Yes that worked thanks a Lot!
